libs/article-parser/src/lib/parser/ebi-parser.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ParsedArticle,
+  Parser,
+  ParsedArticleParagraph,
+  EbiParserOptions,
+} from '@foodmedicine/interfaces';
+import * as cheerio from 'cheerio';
+
+/**
+ * A parser for https://www.ebi.ac.uk/europepmc/webservices/rest/
+ */
+export const EbiParser: Parser<ParsedArticle> = {
+  parserF: async (xml: string, opts?: EbiParserOptions) => {
+    if (
+      !opts ||
+      !opts.parsedArticleHead ||
+      !opts.impacted ||
+      !opts.recommendation
```

Comment:
Nit: You can shorten this a bit using TS optional chaining:
`if (!opts?.parsedArticleHead || !opts?.impacted || !opts?.recommendation) {...}`

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining